tests/runtime/out_stackdriver.c

```diff
@@ -1585,6 +1584,49 @@ void flb_test_resource_global()
     flb_destroy(ctx);
 }
 
+void flb_test_resource_global_custom_prefix()
+{
+    /* configuring tag_prefix for non-k8s resource type should have no effect at all */
+    int ret;
+    int size = sizeof(JSON) - 1;
+    flb_ctx_t *ctx;
+    int in_ffd;
+    int out_ffd;
+
+    /* Create context, flush every second (some checks omitted here) */
+    ctx = flb_create();
+    flb_service_set(ctx, "flush", "1", "grace", "1", NULL);
+
+    /* Lib input mode */
+    in_ffd = flb_input(ctx, (char *) "lib", NULL);
+    flb_input_set(ctx, in_ffd, "tag", "test", NULL);
+
+    /* Stackdriver output */
+    out_ffd = flb_output(ctx, (char *) "stackdriver", NULL);
+    flb_output_set(ctx, out_ffd,
+                   "match", "test",
+                   "google_service_credentials", SERVICE_CREDENTIALS,
+                   "resource", "global",
+                   "tag_prefix", "custom_tag",
+                   NULL);
+
+    /* Enable test mode */
+    ret = flb_output_set_test(ctx, out_ffd, "formatter",
+                              cb_check_global_resource,
```

Comment:
this callback check applies for this case?

Reply:
This test is to cover the case that if the `tag_prefix` is set for the non-k8s resource type (e.g. global, gce_instance), it should have no effect. So the callback is the same.

Reply:
The test is to test the case that if the tag_prefix is added to the non-k8s resource type. The tag_prefix should have no effect at all in this case. So I used this callback.